plugins/kubernetes/app/models/watchers/cluster_pod_error_watcher.rb

```diff
@@ -0,0 +1,35 @@
+module Watchers
+  class ClusterPodErrorWatcher < BaseClusterWatcher
+    include Celluloid::Notifications
+
+    def initialize(client)
+      @client = client
+      super(@client.watch_events(field_selector: 'involvedObject.kind=Pod'))
+    end
+
+    private
+
+    def handle_notice(notice)
+      if error_notice?(notice.object.reason)
+        rc_name = rc_name(notice.object.involvedObject)
+        publish(rc_name, notice) if rc_name
```

Comment:
In here we need to publish to the project topic. I saw you already adapted the ClusterPodWatcher to do so, we just need to do the same here.